### ISSUE TITLE
darkino.online

### ISSUE THREAD

**Issue by mooms06** [easylist/listefr#159]
Sur `https://www2.darkino.online` on ne peut plus obtenir les liens de téléchargement sans désactiver son bloqueur

Exemple:
`
https://www2.darkino.online/download/12757031`

Avec uBlock le bouton "obtenir le lien" n'aboutit pas, si on désactive uBlock ça fonctionne.

**Comment by mapx-** [comment]
The filter added today does not work for you ?
https://github.com/uBlockOrigin/uAssets/commit/67d79980b3fa38c8644c43e9c0b4b4231c98fcb8

@smed79 you added the filters in that list dedicated to warnings

**Comment by mapx-** [comment]
in firefox seems not working

**Comment by mooms06** [comment]
@mapx- Yes, I have the same behavior, working on chromium but not on Firefox.
